sdk/communication/azure-communication-phonenumbers/azure/communication/phonenumbers/_phone_numbers_client.py

```diff
@@ -4,19 +4,35 @@
 # license information.
 # --------------------------------------------------------------------------
 
+from typing import TYPE_CHECKING
 from azure.core.tracing.decorator import distributed_trace
-from azure.core.paging import ItemPaged
-from azure.core.polling import LROPoller
 from ._generated._phone_numbers_client import PhoneNumbersClient as PhoneNumbersClientGen
 from ._generated.models import PhoneNumberSearchRequest
 from ._shared.utils import parse_connection_str, get_authentication_policy
 from ._version import SDK_MONIKER
 
+if TYPE_CHECKING:
```

Comment:
What is the 'if TYPE_CHECKING' for?

Reply:
This is a validation we put for the interpreter to have reference to the types of parameters we declare in our code. It's just type hinting stuff.

Reply:
do we need to add this in all other files?